app/controllers/people_controller.rb

```diff
@@ -0,0 +1,112 @@
+require "#{Rails.root}/lib/scraping/hpi_data_collector.rb"
+require "#{Rails.root}/lib/scraping/scraping_exception.rb"
+
+class PeopleController < ApplicationController
+  before_action :set_person, only: %i[show edit update destroy]
+  http_basic_authenticate_with name: "red", password: "secret"
```

Comment:
Big TODO:
use rails credentials in the future: https://www.codewithjason.com/understanding-rails-secrets-credentials/

Reply:
Also not work of this PR, but I'm a fan of writing a TODO comment :D